assets/components/featuredDigitalPack/featuredDigitalPack.jsx

```diff
@@ -19,6 +23,7 @@ type PropTypes = {
   headingSize: HeadingSize,
   countryGroupId: CountryGroupId,
   url: string,
+  abTest: ComponentAbTest | void,
```

Comment:
flow question: why isn't this modelled as a maybe type?

Reply:
Flow's [maybe type](https://flow.org/en/docs/types/maybe/) allows three types to leak through, e.g. `?string` means `string | null | undefined`. I find that most of the time you don't actually *want* both `null` and `undefined`, you typically just want "value or absence of a value". As @rupertbates pointed out above, this should probably actually be `ComponentAbTest | null`, I'm going to change it now.

Reply:
interesting - thanks for the info 👌